json-sports-team/index.js

```diff
@@ -1,178 +1,257 @@
+const teamName = document.getElementById("team");
+const sport = document.getElementById("sport");
+const year = document.getElementById("year");
+const headCoach = document.getElementById("head-coach");
+const matches = document.getElementById("matches");
+const playerCards = document.getElementById("player-cards");
+const selectElement = document.getElementById("players");
+
+/**
+ * I don't know if we want them to build out this JSON data or just supply it to them in the first step.
+ * If we supply them with the myTeam JSON then we can have the first few steps be about reviewing how to access property values.
+ * Ex. console.log(myTeam.Sport);
+ * We could have them review dot and bracket notation.
+ * This will also be good review for nested objects.
+ * Ex. console.log(myTeam["Head Coach"].Name);
+ */
+
 const myTeam = {
```

Comment:
It might be best for most of this object to follow more typical conventions, with object keys in camelCase except for one to show when bracket notation is necessary:

```
const myTeam = {
  team: "Argentina",
  sport: "Football",
  year: 1986,
  isWorldCupWinner: true,
  "head coach": {
    ...
```